tests/ProxyManagerTest/ProxyGenerator/AccessInterceptorScopeLocalizer/MethodGenerator/BindProxyPropertiesTest.php

```diff
@@ -69,30 +69,16 @@ public function testBodyStructure(): void
         );
 
         $expectedCode = <<<'PHP'
-$this->publicProperty0 = & $localizedObject->publicProperty0;
-
-$this->publicProperty1 = & $localizedObject->publicProperty1;
-
-$this->publicProperty2 = & $localizedObject->publicProperty2;
-
-$this->protectedProperty0 = & $localizedObject->protectedProperty0;
-
-$this->protectedProperty1 = & $localizedObject->protectedProperty1;
-
-$this->protectedProperty2 = & $localizedObject->protectedProperty2;
-
-\Closure::bind(function () use ($localizedObject) {
-    $this->privateProperty0 = & $localizedObject->privateProperty0;
-}, $this, 'ProxyManagerTestAsset\\ClassWithMixedProperties')->__invoke();
-
-\Closure::bind(function () use ($localizedObject) {
-    $this->privateProperty1 = & $localizedObject->privateProperty1;
-}, $this, 'ProxyManagerTestAsset\\ClassWithMixedProperties')->__invoke();
-
-\Closure::bind(function () use ($localizedObject) {
-    $this->privateProperty2 = & $localizedObject->privateProperty2;
-}, $this, 'ProxyManagerTestAsset\\ClassWithMixedProperties')->__invoke();
-
+$class = new \ReflectionObject($localizedObject);
+$this->bindProxyProperty($localizedObject, $class, 'publicProperty0');
```

Comment:
It would be better to inline these, instead of generating `$this->bindProxyProperty()`.

Remember that this is generated code, so we don't really want to abstract it, but rather let the optimizer do as much work as possible.

In additon to that, `bindProxyProperty` is a too generic name, and will likely collide with parent class implementation.

Reply:
Done